### PR TITLE
ACM-15021 ACM-15043 Update documentation links [release-2.12]

### PR DESCRIPTION
**Pull request by KevinFCormier**
Addresses documentation link updates from:
https://issues.redhat.com/browse/ACM-15043
https://issues.redhat.com/browse/ACM-15021

### CODE REVIEW

File: frontend/src/lib/doc-util.tsx [hunk 1]
```diff
@@ -51,16 +51,19 @@ export const DOC_LINKS = {
...
-  HYPERSHIFT_MANAGE_KUBEVIRT: `${DOC_BASE_PATH}/clusters/index#hosted-control-planes-manage-kubevirt`,
-  HOSTED_ENABLE_FEATURE_AWS: `${DOC_BASE_PATH}/clusters/index#enable-or-disable-hosted-control-planes`,
-  HYPERSHIFT_OIDC: `${DOC_BASE_PATH}/clusters/index#hosted-create-aws-secret`,
-  GITOPS_CONFIG: `${DOC_BASE_PATH}/applications/index#gitops-overview`,
+  GITOPS_CONFIG: `${DOC_BASE_PATH}/gitops/gitops-overview`,
   DEPRECATIONS_AND_REMOVALS: `${DOC_BASE_PATH}/release_notes/release-notes#deprecations-removals`,
   ENABLE_OBSERVABILITY: `${DOC_BASE_PATH}/observability/observing-environments-intro#enabling-observability-service`,
   ACCESSING_CONSOLE: `${DOC_BASE_PATH}/web_console/web-console#accessing-your-console`,
-  ROSA_DISCOVERY_AUTOIMPORT_POLICY_EXAMPLE: `${DOC_BASE_PATH}/clusters/cluster_mce_overview#import-discover-rosa`,
+  ROSA_DISCOVERY_AUTOIMPORT_POLICY_EXAMPLE: `${DOC_BASE_PATH}/multicluster_engine_operator_with_red_hat_advanced_cluster_management/mce-acm-integration#import-discover-rosa`,
```
Comment (dockerymick) [inline frontend/src/lib/doc-util.tsx:58]: We should change `multicluster_engine_operator_with_red_hat_advanced_cluster_management/mce-acm-integration` to `mce_acm_integration/acm_integrate_import_rosa`

Comment (KevinFCormier) [reply frontend/src/lib/doc-util.tsx:58]: @dockerymick Why do you think that the path will change compared to https://content-stage.docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.12/html-single/multicluster_engine_operator_with_red_hat_advanced_cluster_management/mce-acm-integration#import-discover-rosa when the docs become available in production without the `content-stage` subdomain?

Is `content-stage.docs.redhat.com` out-of-date? If so, how often is it updated from our docs?

Comment (dockerymick) [reply frontend/src/lib/doc-util.tsx:58]: nvm, I made a comment in the open PR.

**Pull request merged**